### ISSUE TITLE
MacVim releases download URL

### ISSUE THREAD

**Issue by craigrichardson** [autopkg/jleggat-recipes#59]
Hi There,

I think the download details for MacVim have changed.

https://github.com/macvim-dev/macvim/releases

Cheers,
Craig

**Comment by merced317** [comment]
I too would like to use this recipe but can't until the download URL gets updated as suggested.

**Comment by gregneagle** [comment]
Why can't you just adjust your override to change REPO_OWNER to "macvim-dev" for now?

This is one of the reasons overrides _exist_, and more specifically, why many download details are exposed as overridable variables.

**Comment by gregneagle** [comment]
Alternately, since jleggat hasn't updated the recipe in almost a year, you could decide he's no longer interested and make your own. It's odd to act like you can't do _anything_ until the original author performs work for you.

**Comment by merced317** [comment]
You are correct on all accounts. My comment was worded such that it sounded like I was unable to do any of my own work to make the recipe work for me. I will take your advice and attempt to override the recipe. Thank you for your assistance. I still hope this recipe gets updated at some point so others won't have to maintain their own overrides. I do not wish to be a maintainer of this recipe, just a gracious consumer.
